package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "tictactoegit",
+  "version": "1.0.0",
+  "description": "Simple Game",
+  "main": "index.js",
+  "scripts": {
+    "start": "./node_modules/.bin/live-server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Kesse8401/TicTacToe.git"
+  },
+  "author": "Agnes Nowak",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Kesse8401/TicTacToe/issues"
+  },
+  "homepage": "https://github.com/Kesse8401/TicTacToe#readme",
+  "dependencies": {
```

Comment:
Ah, almost missed this one.
You were supposed to add a dev dependency. Change `dependencies` to `devDependencies`.
I will ask you at home about the difference between the two.

Reply:
All about one (mentioned above in the comments are done)